OlinkAnalyze/R/Olink_heatmap.R

```diff
@@ -151,6 +153,26 @@ olink_heatmap_plot <- function(df,
                        values_from = NPX) %>%
     tibble::column_to_rownames('SampleID')
 
+  # Label columns according to the colnames argument
+  if(!colnames %in% c('Assay', 'oid', 'both')){
+    stop('colnames has to be \'Assay\', \'oid\', or \'both\'')
+  }
+  if(colnames == 'Assay'){
+    assays <- sub(pattern = '(.*)_.*', replacement = '\\1', x = colnames(npxWide)) %>%
```

Comment:
Are we sure enough that there will not be a underscore in protein names ever? Would a pattern matching on "OID" be more resilient?

Reply:
Good point! Since the matching is greedy, I think it would still work. But I added a pattern matching for OID anyway to be on the safe side